### ISSUE TITLE
Can you classify an arbitrary song with Model.final?

### ISSUE THREAD

**Issue by laag115** [mlachmish/MusicGenreClassification#2]
If I download a new song that is not in the Dataset, how could I see what genre it is using the Model.final output file?

**Comment by pl425** [comment]
??

**Comment by eggfly** [comment]
Use preproccess.py to load a mp3 or .au etc.. to mfcc or mel-spec data, then start a tensorflow session, put some code like this:
`
saver.restore(sess, "../output/model.final")
xs,  ys = /* your  preproccessed data */
acc = sess.run(accuracy, feed_dict={x: xs, y: ys, keep_prob: 1.})
loss = sess.run(cost, feed_dict={x: xs, y: ys, keep_prob: 1.})
print("Minibatch Loss= " + \"{:.6f}".format(loss) + ", Training Accuracy= " + "{:.5f}".format(acc))
`
when accuracy is 1 then model prediction is right, otherwise accuracy is zero.

**Comment by jqzhao7** [comment]
I'm sorry to disturb you. Since the link of the data set of this project in the current year is invalid and the author cannot be contacted, I can only ask you in this way. If you still have the data set of this project, could you please send it to me [zjq2048@qq.com](mailto:zjq2048@qq.com)
